lib/skin.js

```diff
@@ -27,29 +36,100 @@ module.exports = function () {
     }
   }
 
-  function createSkin(skinId, baseResponsiveSkins) {
-    var skinDir = path.resolve('res/skins', skinId);
-    return new Skin(skinId, skinDir, baseResponsiveSkins);
+  function createSkin(skinId, skinDir) {
+    return new Skin(skinId, skinDir);
   }
 
-  var coreResponsiveSkinIds = ['responsive_peak'];
+  var coreResponsiveSkinIds = ['responsive_peak', 'responsive_base'];
```

Comment:
maybe replace this with logic to get all the core responsive skins we downloaded from core?